environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentResponseConverter.java

```diff
@@ -219,8 +232,8 @@ private ResourceGroupUsage resourceGroupUsagePatternToResourceGroupUsage(Resourc
                 return ResourceGroupUsage.MULTIPLE;
             case USE_SINGLE_WITH_DEDICATED_STORAGE_ACCOUNT:
                 return ResourceGroupUsage.SINGLE_WITH_DEDICATED_STORAGE_ACCOUNT;
-                default:
-                    throw new RuntimeException("Unknown resource group usage pattern: %s" + resourceGroupUsagePattern);
+            default:
+                throw new RuntimeException("Unknown resource group usage pattern: %s" + resourceGroupUsagePattern);
```

Comment:
I know this is not new code, but why is this a Runtime exception?
I think we should use an exception that indicates that the provided value illegal so maybe bad request should be used here if this is only a request time validation or an illegalstate exception otherwise.